apps/heft/.npmignore

```diff
@@ -6,13 +6,10 @@
 !/lib/**
 !/dist/**
 !ThirdPartyNotice.txt
-!/EULA/**
 
 # Ignore certain files in the above folder
 /dist/*.stats.*
-/lib/**/test/**
-/lib/**/*.js.map
-/dist/**/*.js.map
+/lib/**/test/*
```

Comment:
Interesting. What if the test files are split into multiple categories? Some test folders contain folders of test-related assets, would these be included now?

Reply:
The rule is correct, but I think it's easier to see that it's correct if we write it as `/lib/**/test/`.  I also changed the first rule from `**` to `*` (also equivalent, but easier to understand).